verification_token/models.py

```diff
@@ -104,8 +104,7 @@ def generate_key(cls, generator=None, *args, **kwargs):
     @property
     def is_valid(self):
         return (
-            self.is_active and self.key and (self.expiration_in_minutes is None or timezone.now() <=
-                                             self.created_at + timedelta(minutes=self.expiration_in_minutes))
+            self.is_active and self.key and (self.expires_at is None or timezone.now() <= self.expires_at)
```

Comment:
Here, `localtime` is not needed but I would unify things and use it here as well.